File: Initial Prototype/ServerSide/FindNDrive/DomainObjects/Domains/GeoAddress.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="GeoAddress.cs" company="">
//   
// </copyright>
// <summary>
//   The geo address.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace DomainObjects.Domains
{
    using System.ComponentModel.DataAnnotations;
    using System.Runtime.Serialization;

    /// <summary>
    /// The geo address.
    /// </summary>
    [DataContract]
    public class GeoAddress
    {
        /// <summary>
        /// Gets or sets the geo address id.
        /// </summary>
        [DataMember]
        [ConcurrencyCheck]
        public int GeoAddressId { get; set; }

        /// <summary>
        /// Gets or sets the order.
        /// </summary>
        [DataMember]
        public int Order { get; set; }

        /// <summary>
        /// Gets or sets the address line.
        /// </summary>
        [DataMember]
        public string AddressLine { get; set; }

        /// <summary>
        /// Gets or sets the latitude.
        /// </summary>
        [DataMember]
        public double Latitude { get; set; }

        /// <summary>
        /// Gets or sets the longitute.
        /// </summary>
        [DataMember]
        public double Longitude { get; set; }
    }
}

File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/domain_objects/GeoAddress.java
package com.example.myapplication.domain_objects;

/**
 * Created by Michal on 09/01/14.
 */
public class GeoAddress {
    public double Latitude;
    public double Longitude;
    public int Order;
    public String AddressLine;

    public GeoAddress(double Latitude, double Longitude, String AddressLine, int order)
    {
        this.Latitude = Latitude;
        this.Longitude = Longitude;
        this.AddressLine = AddressLine;
        this.Order = order;
    }
}
